src/openforms/js/lang/formio/nl.json

```diff
@@ -183,5 +183,10 @@
   "Native browser dropdown": "Native browser dropdown",
   "Multiple values": "Meerdere waarden",
   "Are there multiple values possible for this field?": "Zijn er voor dit veld meerdere waarden mogelijk?",
-  "Search field(s)": "Zoek veld(en)"
+  "Search field(s)": "Zoek veld(en)",
+  "Minimum Date": "Minimale datum",
+  "The minimum date that can be picked.": "De minimale datum die gekozen kan worden.",
+  "Maximum Date": "Maximale datum",
+  "The maximum date that can be picked.": "De maximale datum die gekozen kan worden.",
+  "invalid_time": "Alleen tijden tussen {{ minTime }} en {{ maxTime }} zijn toegestaan."
```

Comment:
I'm missing the translations for the newly added titles/tooltips in `components/form/time.js`